### PR TITLE
Add Turkish translation

### PR DESCRIPTION
**Pull request by yakushabb**

### CODE REVIEW

File: po/tr.po [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+msgstr ""
+"Flood It, mümkün olan en az hamlede tüm panoyu tek renkle doldurmanızı "
+"isteyen eğlenceli bir oyundur. Basit fakat bağımlılık yapan strateji "
+"oyunuyla kendinize meydan okurken olabildiğince verimli şekilde panoyu "
+"renklere boyayın!"
+
+#. Translators: These are search terms to find this application. Do NOT translate or localize the semicolons. The list MUST also end with a semicolon.
+#: data/io.github.tfuxu.floodit.desktop.in.in:12
+msgid "game;strategy;flood;flood-it;"
+msgstr "game;strategy;flood;flood-it;oyun;strateji;sel;baskın;taşkın;"
```
Comment (tfuxu) [inline po/tr.po:47]: Shouldn't keywords in English be removed?

**Review by tfuxu — CHANGES_REQUESTED** [review]

File: po/tr.po (hunk 1 above)
Comment (yakushabb) [reply po/tr.po:47]: Due to a glib error, translators should keep the existing keywords intact and add new keywords afterward.

Comment (tfuxu) [reply po/tr.po:47]: I see, thanks for clarification! BTW, to you have a link to the issue?

Comment (yakushabb) [reply po/tr.po:47]: https://gitlab.gnome.org/GNOME/glib/-/issues/443

**Review by tfuxu — APPROVED** [review]

**Pull request merged**